sound/soc/sof/intel/pci-skl.c

```diff
@@ -52,6 +55,9 @@ static struct sof_dev_desc kbl_desc = {
 	.default_fw_path = {
 		[SOF_INTEL_IPC4] = "intel/avs/kbl",
 	},
+	.default_lib_path = {
+		[SOF_INTEL_IPC4] = "intel/sof-lib/kbl",
+	},
```

Comment:
wondering if the library loading is the same for SKL/KBL, it's not the same loader at all. Maybe we should put this on the back-burner for now, it's not clear how we would test this and it's not really a priority, is it?

Reply:
OK, I'll drop this and the setting of the load_library callback for SKL.